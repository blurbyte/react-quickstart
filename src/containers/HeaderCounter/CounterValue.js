import styled from 'styled-components';

const CounterValue = styled.span`
  padding: 0 0.4rem;
  border-radius: 6px;
  display: block;
  min-width: 6rem;
  height: 2.5rem;
  line-height: 2.5rem;
  background: #ff4326;
  font-family: 'OpenSans Bold', Arial, Helvetica, sans-serif;
  text-align: center;
  font-size: 1.8rem;
  color: #fff;
  cursor: default;
`;

export default CounterValue;
