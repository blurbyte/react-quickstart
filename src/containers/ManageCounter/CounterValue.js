import styled from 'styled-components';

const CounterValue = styled.p`
  padding: 0 0.4rem;
  border-radius: 6px;
  font-family: 'OpenSans Bold', Arial, Helvetica, sans-serif;
  font-size: 2.4rem;
  line-height: 3.4rem;
  margin: 1rem 0;
  color: #fff;
  background: #ff4326;
  display: inline-block;
  min-width: 6rem;
  text-align: center;
`;

export default CounterValue;
