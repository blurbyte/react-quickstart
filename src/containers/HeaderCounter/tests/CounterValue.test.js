import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import CounterValue from '../CounterValue';

describe('<CounterValue />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(<CounterValue>10</CounterValue>);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
