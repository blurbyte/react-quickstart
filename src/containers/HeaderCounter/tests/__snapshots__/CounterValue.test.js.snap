// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CounterValue /> should render correctly 1`] = `
.fSvMUz {
  padding: 0 0.4rem;
  border-radius: 6px;
  display: block;
  min-width: 6rem;
  height: 2.5rem;
  line-height: 2.5rem;
  background: #ff4326;
  font-family: 'OpenSans Bold', Arial, Helvetica, sans-serif;
  text-align: center;
  font-size: 1.8rem;
  color: #fff;
  cursor: default;
}

<span
  className="fSvMUz"
>
  10
</span>
`;
